### PR TITLE
IJPS-117: Calculate Published sitting record count

### PR DESCRIPTION
**Pull request by SomeshRA**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/IJPS-117

### Change description ###
 Calculate Published sitting record count

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: src/integrationTest/resources/sql/insert_published_test_data.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
+INSERT INTO public.sitting_record ( sitting_date, status_id, region_id, epimms_id, hmcts_service_id, personal_code, contract_type_id, judge_role_type_id, am, pm)
+VALUES ( '2023-05-11', 'SUBMITTED', '3', '852649', 'BBA3', '4918178', 6, 'HealthWorker', false, true);
```
Comment (markdathornehmcts) [inline src/integrationTest/resources/sql/insert_published_test_data.sql:2]: Maybe use enum values (e.g. status, judge role type, contract type)

File: src/main/java/uk/gov/hmcts/reform/jps/repository/SittingRecordRepository.java [hunk 2]
```diff
@@ -46,4 +46,18 @@ public interface SittingRecordRepository extends JpaRepository<SittingRecord, Lo
         + " where s.id = :sittingRecordId and s.statusId = 'RECORDED'"
```
Comment (markdathornehmcts) [inline src/main/java/uk/gov/hmcts/reform/jps/repository/SittingRecordRepository.java:46]: Maybe use enum value of status RECORDED

File: src/main/java/uk/gov/hmcts/reform/jps/services/SittingRecordService.java [hunk 3]
```diff
@@ -336,4 +341,58 @@ private String getAccountCode(String hmctsServiceCode) {
...
+            .build();
+
+        LocalDate previousYear = currentDate.minusYears(1L);
+        String previousFinancialYear = getFinancialYear(previousYear);
+        FinancialYearRecords previousFinancialYearRecords = FinancialYearRecords.builder()
+            .publishedCount(sittingDaysService.getSittingCount(personalCode, previousFinancialYear))
+            .submittedCount(
+                sittingRecordRepository.findCountByPersonalCodeAndStatusIdAndFinancialYearBetween(
+                        personalCode,
+                        SUBMITTED,
```
Comment (markdathornehmcts) [inline src/main/java/uk/gov/hmcts/reform/jps/services/SittingRecordService.java:367]: May be enum value of status SUBMITTED

**Review by markdathornehmcts — APPROVED** [review]
Approved with. few suggestions

File: src/main/java/uk/gov/hmcts/reform/jps/services/SittingRecordService.java (hunk 3 above)
Comment (SomeshRA) [reply src/main/java/uk/gov/hmcts/reform/jps/services/SittingRecordService.java:367]: Its the enum from StatusId

**Pull request merged**